templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,32 @@
 {
```

Comment:
<p>Is the goal to replace all of this with Mantis once we have the single directory saving implemented? Or some of these plots should be kept even once we make that change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
If I understand it correctly, the mantis function (create_mantis_dir) just copies all the masks over, so the single directory saving will just take these masks and copy them over. This function actually creates the overlays.